src/styles/images/index.md.njk

```diff
@@ -8,53 +8,70 @@ layout: layout-pane.njk
 
-### Illustrations or representative imagery
+### Alt text
 
-If your image represents something physical, such as a letter, document or credit card you should use the aspect ratio of that item.
+Alternative text, or alt text, is read out by screen readers or displayed if an image does not load or if images have been switched off.
 
-{{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
+All images should include the alt attribute, even if it’s left empty without alt text (`alt=""`).
 
+#### When to use alt text
 
-## Alternative text
+Only add alt text when you need to describe visual details that cannot practically be included in the written content.
 
-This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
+There are some cases when an empty `alt=""` attribute is more appropriate. These include:
 
-Alternative text, or alt text, is read out by screen readers or displayed if an image does not load or if images have been switched off.
+- a decorative image that does not include important content
+- an icon with a text label, where the alt text would repeat the text label
+- an image used in a link, where the image is not needed to understand the link
+
+To decide if an image needs alt text, see the [Web Accessibility Initiative alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).
 
-All images, except decorative images, must have alt text that:
+#### Writing good alt text
+
+Good alt text:
 
 - tells people what information the image provides
 - describes the content and function of the image
 - is specific, meaningful and concise
 
 Use normal punctuation, like commas and full stops, so the text is easy to read and understand.
 
-Do not:
+Generally, aim for no more than 2 sentences of alt text.
+
+Do not write alt text that:
 
-- include the name of the photographer or person who created the image
-- start with ‘Image of’, ‘Graphic of’ or ‘Photo of’
-- repeat information from the page
-- include extra information not in the image
+- includes the name of the photographer or person who created the image
+- starts with ‘Image of’, ‘Graphic of’ or ‘Photo of’
+- repeats information from the page
+- includes extra information not in the image
 
 {{ example({group: "styles", item: "images", example: "alt-text", html: true, open: true, size: "l"}) }}
 
-Some images do not need descriptions. Use `alt=""` if the image is:
+Avoid images that contain text, especially PNGs and JPGs because the text becomes unreadable when the user zooms in. If the image does contain text, use SVG because it remains readable when the user zooms in. The alt text should be the same as the text displayed in the image.
```

Comment:
Could this paragraph use a heading of its own? (Like 'Avoid images that contain text', and then rejig the first sentence.) Just so users don't miss it, what with it being under the image.

Reply:
Oh, that's interesting... semantically it'd fall under ## Using images to give information.

```suggestion
### Avoid images that contain text

Text becomes unreadable when the user zooms into images, especially PNGs and JPGs. If the image does contain text, use SVG because it remains readable when the user zooms in. The alt text should be the same as the text displayed in the image.
```

Reply:
Nice!